invoicer_project/clients/urls.py

```diff
@@ -1,13 +1,15 @@
 from django.urls import path
-from .views import home_view, ClientViewSet
+from .views import ClientViewSet
+from django.views.generic import TemplateView
 
 clients_methods = ClientViewSet.as_view({'post': 'create', 'get': 'list'})
 clients_list = ClientViewSet.as_view({"get": "list"})
 delete_client = ClientViewSet.as_view({'delete': 'destroy'})
 
 urlpatterns = [
-    path("home/", home_view, name='home'),
+    path("home/", TemplateView.as_view(template_name="clients/clients_list.html"), name='home'),
     path("list/", clients_list, name='clients'),
     path("client/<client_id>", delete_client, name='delete_client'),
```

Comment:
it is usefull to restrict parameter client_id to int
```suggestion
    path("client/<int:id>", delete_client, name='delete_client'),
```